pkg/api/componentreadiness/component_report.go

```diff
@@ -1345,16 +1345,20 @@ func (c *componentReportGenerator) generateComponentTestReport(baseStatus map[st
 		if !ok {
 			testStats.ReportStatus = crtype.MissingSample
 		} else {
-			var approvedRegression *regressionallowances.IntentionalRegression
+			var approvedRegression, baseRegression *regressionallowances.IntentionalRegression
 			if len(c.VariantCrossCompare) == 0 { // only really makes sense when not cross-comparing variants:
 				// look for corresponding regressions we can account for in the analysis
 				approvedRegression = regressionallowances.IntentionalRegressionFor(c.SampleRelease.Release, testID.ColumnIdentification, testID.TestID)
-				resolvedIssueCompensation, triagedIncidents = c.triagedIncidentsFor(testID)
+				baseRegression = regressionallowances.IntentionalRegressionFor(c.BaseRelease.Release, testID.ColumnIdentification, testID.TestID)
+				// ignore triage if we have an intentional regression
+				if approvedRegression == nil {
+					resolvedIssueCompensation, triagedIncidents = c.triagedIncidentsFor(testID)
+				}
```

Comment:
I don't notice any time window used here - how does it work if I were to select a different time value for 4.17 where the regression wasn't present?

Don't we want this to only work in the GA window case?

Reply:
The IntentionalRegressions are expected to be added at the end of the release during the GA timeframe and lower the bar for the entire release.  That's not new with this PR.  We could build your case out if we needed but I think that is outside of what this PR is trying to do.

When adding the [base regression override](https://github.com/openshift/sippy/blob/f1363e049ed106202f2a99432af68f69dc9a79d4/pkg/api/componentreadiness/component_report.go#L1588) it should only kick in when the computed base values are less than the previous values.  So if we see the base improve we won't then lower to the previous values.

Reply:
I don't think that's great protection and going to result in confusion.  We won't really be able to answer questions like "what was 4.17 like in the middle of the development cycle? At the beginning?" Or even slice the last month of GA because this is such a big hammer.